src/pytorch_lightning/utilities/imports.py

```diff
@@ -42,6 +42,7 @@
 _RICH_AVAILABLE = package_available("rich") and compare_version("rich", operator.ge, "10.2.2")
 _TORCH_QUANTIZE_AVAILABLE = bool([eg for eg in torch.backends.quantized.supported_engines if eg != "none"])
 _TORCHVISION_AVAILABLE = package_available("torchvision")
+_TORCHVISION_GREATER_EQUAL_0_14 = compare_version("torchvision", operator.ge, "0.14.0")
```

Comment:
Could we move this to the file where it is used directly? We try to move towards more localized definitions, see #11826.

Reply:
Thanks a lot for the feedback. I've moved `_TORCHVISION_GREATER_EQUAL_0_14` to `model_helpers.py`. Happy to make any other changes needed. :)

Reply:
Great, thank you!